app/views/play/level/WebSurfaceView.coffee

```diff
@@ -35,6 +38,7 @@ module.exports = class WebSurfaceView extends CocoView
     html = _.find(dom, name: 'html') ? {name: 'html', attribs: null, children: [body]}
     # TODO: pull out the actual scripts, styles, and body/elements they are doing so we can merge them with our initial structure on the other side
     { virtualDom, styles, scripts } = @extractStylesAndScripts(@dekuify html)
+    @extractCssSelectors(styles, e.html)
```

Comment:
Should be `extractCSSSelectors`... but that does look a bit weird, 3 S's in a row.